src/main/java/pacApp/pacConfig/WebSecurityConfig.java

```diff
@@ -99,6 +99,10 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 .antMatchers("/auth/**").permitAll()
                 .antMatchers("/registration").permitAll()
                 .antMatchers("/h2-console/*").permitAll()
+                .antMatchers("/swagger-ui**").permitAll()
+                .antMatchers("/webjars/**").permitAll()
+                .antMatchers("/swagger-resources/**").permitAll()
+                .antMatchers("/v2/api-docs/**").permitAll()
```

Comment:
This is fine for now, but we need some kind of auth for swagger in the future, please think of something @philsch91 or @erdoganeren